astroquery/esa/xmm_newton/core.py

```diff
@@ -433,5 +434,78 @@ def get_epic_images(self, filename, *, band=[], instrument=[],
 
         return ret
 
+    def get_epic_metadata(self, *, target_name=None,
+                          coordinates=None, radius=None):
+        """Downloads the European Photon Imaging Camera (EPIC)
+        metadata from a given target
+
+        Parameters
+        ----------
+        target_name : string, optional, default None
+            The name of the target
+        coordinates : `~astropy.coordinates.SkyCoord`, optinal, default None
+            The coordinates of the target in a SkyCoord object
+        radius : float, optional, default None
+            The radius to query the target in degrees
```

Comment:
There has to be an empty line in the docstring between the sections to be parsed/rendered properly

```suggestion
            The radius to query the target in degrees
            
```